src/utils/sendPaginatedList.js

```diff
@@ -15,11 +16,20 @@ async function sendPaginatedList(res, Model, filter, pageNo, limit) {
     if (limit * pageNo > filteredProductCount) {
       return sendFailedResponse(res, 'It is not possible to split into so many elements');
     }
-    const filteredProduct = await Model.find(filter).skip((pageNo - 1) * limit).limit(limit);
+    const filteredProduct = await Model.find(filter, { __v: 0 }).skip((pageNo - 1) * limit).limit(limit).select(select);
```

Comment:
Change `product` to `item` in all variables name in this file, because each time this method is being called, items are different. One time it maybe product, another time it will be category, and so on